base/src/main/java/com/smartdevicelink/managers/screen/choiceset/PreloadChoicesOperation.java

```diff
@@ -227,14 +230,44 @@ private CreateInteractionChoiceSet choiceFromCell(ChoiceCell cell){
 	}
 
 	// HELPERS
+	boolean shouldSendChoiceText() {
```

Comment:
Is there a reason why this methods are `shouldSend____Text` instead of something more like `is___TextSupported`?

Reply:
No reason other than following the iOS naming to make them easier to compare and maintain in the future. If you prefer changing the names, I suggest removing these 5 methods and call `templateSupports___Field()` directly.

Reply:
If this matches iOS then we can keep these for now. I would like to revisit for both after this release if possible. The naming doesn't make sense in my opinion.